### DISCUSSION TITLE
Select Week ➔ Week picker ?

### DISCUSSION THREAD

**Discussion by ImamJanjua** [gpbl/react-day-picker#2136] [Ideas]
Is this possible? Like it dosen't need to be like something crazy. Like just a bar which selects 7 or x days and maybe on select return dates in an array. And maybe if in the selects days, some days of next month are included than it may also switch the month.

Complete game changer and definitely not the only one who would use it 🔥🔥

https://github.com/gpbl/react-day-picker/assets/137432044/c10ac5f3-9476-458e-9139-5a0754cbeae7

**Comment by gpbl** [comment]
Hi @ImamJanjua, nice idea. You can see an example of selecting a week here:

https://react-day-picker.js.org/advanced-guides/custom-selections#week-selection

![2024-05-15 5 53 50 PM](https://github.com/gpbl/react-day-picker/assets/120693/1962a213-01c0-40cb-a3b4-0cb93e8beb14)

**Reply by sailor95** [reply]
The link is now 404.

**Reply by gpbl** [reply]
Updated link is https://daypicker.dev/guides/custom-selections#week-selection
